mmaction/__init__.py

```diff
@@ -1,3 +1,28 @@
+import mmcv
+
 from .version import __version__, short_version
 
+
+def digit_version(version_str):
```

Comment:
can this version thing be moved to mmcv?

Reply:
Agree, since it is also used in `mmdet`.

Reply:
Found the `digit_version` has already existed in `mmcv`, so I directly import it here.